csharp-intermediate-topics/TupleAliasInCSharp/TupleAliasInCSharp/TupleAlias.cs

```diff
@@ -0,0 +1,35 @@
+﻿namespace TupleAliasInCSharp
```

Comment:
We can use file-scoped namespaces here as well

Reply:
This PR has been updated to reflect recent changes.